source/renderer/app/components/news/AlertsOverlay.js

```diff
@@ -58,15 +59,21 @@ export default class AlertsOverlay extends Component<Props, State> {
     this.props.onMarkNewsAsRead(alerts[0].date);
   };
 
+  onProceedNewsAction(event: SyntheticMouseEvent<HTMLElement>) {
+    const { onProceedNewsAction, alerts } = this.props;
+    const [alert] = alerts;
```

Comment:
There are `const [alert] = alerts;` and `alerts[0]` in many places. Maybe create a `get alert() {return this.props.alerts[0]}` method?